plugin/lspconfig.lua

```diff
@@ -135,7 +135,7 @@ api.nvim_create_user_command('LspStop', function(info)
   end
 
   if not server_id then
-    local servers_on_buffer = require('lspconfig.util').get_lsp_clients { bufnr = current_buf }
+    local servers_on_buffer = vim.lsp.get_clients { bufnr = current_buf }
```

Comment:
this only support 0.10 and the util wrap also support 0.9

Reply:
OK why is the method missing though?

Reply:
post the error you get first and you env etc.

Reply:
```
Error executing vim.schedule lua callback: /usr/share/nvim/runtime/lua/vim/lsp.lua:374: Error executing lua: vim/keymap.lua:0: E31: No such mapping
stack traceback:
        [C]: in function 'nvim_buf_del_keymap'
        vim/keymap.lua: in function 'del'
        /usr/share/nvim/runtime/lua/vim/lsp.lua:377: in function </usr/share/nvim/runtime/lua/vim/lsp.lua:374>
        [C]: in function 'nvim_buf_call'
        /usr/share/nvim/runtime/lua/vim/lsp.lua:374: in function 'reset_defaults'
        /usr/share/nvim/runtime/lua/vim/lsp.lua:403: in function </usr/share/nvim/runtime/lua/vim/lsp.lua:389>
stack traceback:
        [C]: in function 'nvim_buf_call'
        /usr/share/nvim/runtime/lua/vim/lsp.lua:374: in function 'reset_defaults'
        /usr/share/nvim/runtime/lua/vim/lsp.lua:403: in function </usr/share/nvim/runtime/lua/vim/lsp.lua:389>
```

```
NVIM v0.10.0
Build type: Release
LuaJIT 2.1.1713773202
```

Arch linux

Reply:
Sorry that was the other LspStop bug I'm still trying to resolve 🥲

Here is the error for this one:

```
Error executing Lua callback: ...ocal/share/nvim/lazy/nvim-lspconfig/plugin/lspconfig.lua:138: attempt to call field 'get_lsp_clients' (a nil value)
stack traceback:
        ...ocal/share/nvim/lazy/nvim-lspconfig/plugin/lspconfig.lua:138: in function <...ocal/share/nvim/lazy/nvim-lspconfig/plugin/lspconfig.lua:125>
```

Reply:
> ```
> Error executing vim.schedule lua callback: /usr/share/nvim/runtime/lua/vim/lsp.lua:374: Error executing lua: vim/keymap.lua:0: E31: No such mapping
> stack traceback:
>         [C]: in function 'nvim_buf_del_keymap'
>         vim/keymap.lua: in function 'del'
>         /usr/share/nvim/runtime/lua/vim/lsp.lua:377: in function </usr/share/nvim/runtime/lua/vim/lsp.lua:374>
>         [C]: in function 'nvim_buf_call'
>         /usr/share/nvim/runtime/lua/vim/lsp.lua:374: in function 'reset_defaults'
>         /usr/share/nvim/runtime/lua/vim/lsp.lua:403: in function </usr/share/nvim/runtime/lua/vim/lsp.lua:389>
> stack traceback:
>         [C]: in function 'nvim_buf_call'
>         /usr/share/nvim/runtime/lua/vim/lsp.lua:374: in function 'reset_defaults'
>         /usr/share/nvim/runtime/lua/vim/lsp.lua:403: in function </usr/share/nvim/runtime/lua/vim/lsp.lua:389>
> ```
> 
> ```
> NVIM v0.10.0
> Build type: Release
> LuaJIT 2.1.1713773202
> ```
> 
> Arch linux

this is your keymap config error . check your keymap

Reply:
See next message sorry

Reply:
> Sorry that was the other LspStop bug I'm still trying to resolve 🥲
> 
> Here is the error for this one:
> 
> ```
> Error executing Lua callback: ...ocal/share/nvim/lazy/nvim-lspconfig/plugin/lspconfig.lua:138: attempt to call field 'get_lsp_clients' (a nil value)
> stack traceback:
>         ...ocal/share/nvim/lazy/nvim-lspconfig/plugin/lspconfig.lua:138: in function <...ocal/share/nvim/lazy/nvim-lspconfig/plugin/lspconfig.lua:125>
> ```

I can't reproduce please try minimal config first. i think open issue would be better.